health/micro-ui/web/micro-ui-internals/packages/css/src/index.scss

```
@@ -20,4 +20,5 @@
 @import "./pages/employee/mapview.scss";
 @import "./pages/employee/payment.scss";
 @import "./pages/employee/healthdss.scss";
-@import "./pages/employee/mycampaignsnew.scss";
\ No newline at end of file
+@import "./pages/employee/mycampaignsnew.scss";
```

Stylelint (17.6.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)